Sources/Testing/Parameterization/Test.Case.swift

```diff
@@ -156,10 +277,32 @@ extension Test {
 // MARK: - Codable
 
 extension Test.Parameter: Codable {}
-extension Test.Case.Argument.ID: Codable {}
+extension Test.Case.Argument.ID: Codable {
+  public init(from decoder: some Decoder) throws {
+    let container = try decoder.container(keyedBy: CodingKeys.self)
+
+    // The `isStable` property was added after this type was introduced.
+    // Previously, only stable argument IDs were ever encoded, so if we're
+    // attempting to decode one, we can safely assume it is stable.
+    let isStable = try container.decodeIfPresent(type(of: isStable), forKey: .isStable) ?? true
+
+    let bytes = try container.decode(type(of: bytes), forKey: .bytes)
+    self.init(bytes: bytes, isStable: isStable)
+  }
+}
 
 // MARK: - Equatable, Hashable
 
+extension Test.Case: Hashable {
```

Comment:
Nitpick: explicitly state `Equatable`.

Is this `==` implementation expensive to call?

Reply:
> Is this == implementation expensive to call?

It primarily amounts to calling `==` on two `Array<UInt8>` operands. It does not trigger (re-)encoding each time, in case that's what you're wondering.